requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 codecov
 coverage
+ipaddress>=1.0.22
```

Comment:
See the message on commit https://github.com/hvac/hvac/pull/347/commits/d0c2961a0d143e3fc7dd4574c8a720c341e96e49 for additional context on this addition...